src/models/orders.ts

```diff
@@ -0,0 +1,40 @@
+import { Asset } from "./commons";
+
+export interface Order {
+    makerAddress: string,
+    makeAsset: Asset,
+    takerAddress: string,
+    takeAsset: Asset,
+    /**
+    * Random number to distinguish between a maker's orders
+    **/
+    salt: number,
+    /**
+    * Order can't be matched before this block timestamp (optional)
+    **/
+    startBlockTimestamp?: number,
+    /**
+    * Order can't be matched after this block timestamp (optional)
+    **/
+    endBlockTimestamp?: number,
+}
+
+export interface OrderFilter {
+    origin: string,
+    sort: 'LAST_UPDATE',
```

Comment:
Does the `sort` field always has the `LAST_UPDATE` value?

Reply:
@carlosfebres This seems to be the case (see [here](https://api-reference.rarible.com/#operation/getSellOrdersByMaker)), currently, but it could change in the future as the API is built out further. Since we don't have the source code for the API, we can only go off the OpenAPI docs and HTTP calls we make to it.